NeuroID/build.gradle

```diff
@@ -106,13 +107,17 @@ dependencies {
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.6.1")
 
     reactNativeLibImplementation("com.facebook.react:react-native:0.71.0-rc.0")
+    implementation 'com.squareup.retrofit2:retrofit:2.9.0'
+    implementation 'com.squareup.retrofit2:converter-gson:2.9.0'
+
     advancedDeviceAndroidLibImplementation("com.fingerprint.android:pro:2.3.2") {
         exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib'
     }
     advancedDeviceReactNativeLibImplementation("com.facebook.react:react-native:0.71.0-rc.0")
     advancedDeviceReactNativeLibImplementation("com.fingerprint.android:pro:2.3.2") {
         exclude group: 'org.jetbrains.kotlin', module: 'kotlin-stdlib'
     }
+    implementation 'com.squareup.okhttp3:okhttp:4.9.2'
```

Comment:
NIT: should we put this next to the retrofit lines to keep them together?

Reply:
grouped together